sdmetrics/single_column/statistical/kscomplement.py

```diff
@@ -56,7 +57,13 @@ def compute(real_data, synthetic_data):
             real_data = pd.to_numeric(real_data)
             synthetic_data = pd.to_numeric(synthetic_data)
 
-        statistic, _ = ks_2samp(real_data, synthetic_data)
+        try:
+            statistic, _ = ks_2samp(real_data, synthetic_data)
+        except ValueError as e:
+            if str(e) == 'Data passed to ks_2samp must not be empty':
```

Comment:
Should we check the inputs instead of trying to catch the error from the error message?

Reply:
Only old versions of scipy raise this error, newer ones support empty data.